main.go

```diff
@@ -169,63 +170,76 @@ func main() {
 const chainLoveURL = "https://api.chain.love/"
 
 func getAnchorPoint(cctx *cli.Context) (*blockGetter, *lchtypes.TipSet, error) {
-	sourceSelect := []string{"car", "tipset-cids", "rpc-endpoint", "rpc-fullnode"}
+	sourceSelect := []string{"car", "rpc-endpoint", "rpc-fullnode"}
 
-	var isSet int
+	var countHeadSources int
 	for _, s := range sourceSelect {
 		if cctx.IsSet(s) {
-			isSet++
+			countHeadSources++
 		}
 	}
-	if isSet == 0 && cctx.Bool("trust-chainlove") {
-		isSet++
+	if countHeadSources == 0 && cctx.Bool("trust-chainlove") {
+		countHeadSources++
 		if err := cctx.Set("rpc-endpoint", chainLoveURL); err != nil {
 			return nil, nil, err
 		}
 	}
 
-	if isSet != 1 {
-		return nil, nil, xerrors.Errorf("you must specify exactly one of: %s", strings.Join(sourceSelect, ", "))
+	if countHeadSources > 1 {
+		return nil, nil, xerrors.Errorf("you can not specify more than one CurrentTipsetSource of: %s", strings.Join(sourceSelect, ", "))
+	} else if countHeadSources == 0 && !cctx.IsSet("tipset-cids") {
+		return nil, nil, xerrors.New("you have not specified any CurrentTipsetSource - you must provide the tipset excplicitly via '--tipset-cids'")
```

Comment:
We're hiding information about the other tipset sources now, right? Shouldn't we expose all of them in the error?

Reply:
Something like that. Although why call out tipset-cids as separate instead of lumping them together with the others?